docs/about/neon_ecosystem.md

```diff
@@ -47,42 +49,26 @@ The Neon Proxy Operator accepts payment from the user in [NEON tokens](/docs/tok
 
 Owners of the Neon token may engage in the Neon DAO activities.
 
-### Neon Operators
+## Neon DAO
 
-**Neon Operators** run Neon Proxy servers to facilitate the seamless execution of Ethereum-like transactions on Solana. It is this proxy service that allows Ethereum dApps to be ported to Neon with no code changes. Operators accept payment in NEON and make payments within the Solana ecosystem in SOL.
+The DAO essentially owns the Neon EVM and is also a grant-issuing authority able to direct future research and development. The Neon DAO exists as a series of contracts deployed on Solana to provide a Layer 0 (L0: a social/governance layer) to control the functionality of Neon EVM. The DAO provides a web interface to allow the community, i.e. NEON holders, to raise and vote on proposals that impact the functioning of the Neon EVM.
```

Comment:
Layer 0 - is it a common way to name DAO? Usually it is used for inter-chain protocols